### DISCUSSION TITLE
記事の保存ボタンは画面下部にあった方が嬉しい

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#511] [要望・アイデア・不具合]
このGitHubのdiscussionフォームもそうですが、長文を入力して保存するタイプの入力フォームは保存ボタンが画面の下部にあることが多いように思います。

<img width="881" alt="github" src="https://github.com/increments/qiita-discussions/assets/1148320/eb8eeae5-8f37-4ced-bf32-15cc38adbd2d">

一方、Qiitaは最近のアップデートで保存ボタンが画面上部に移動しました。

<img width="1211" alt="edit" src="https://github.com/increments/qiita-discussions/assets/1148320/6e7ea2d7-5840-4577-88c3-b3aa0169e1bd">

慣れの問題かもしれませんが、記事を書き終わると、毎回「あれっ、保存ボタンはどこだっけ？」と一瞬ボタンを探してしまいます。
記事を書くときも、「文章を上から下まで読んで・・・よし、保存」とする場合は、一番下に保存ボタンがある方がメンタルモデル的にも自然な気がします。

加えて、保存ボタンの色が白色で目立たないのも「あれっ、どこだっけ？」に拍車をかけてる印象です。

ボタンを上に置いたのはスマホ入力時にテキストエリアを広く使うためかな？、と推測したりしますが、個人的にはパソコン上で記事を編集することが多いので、保存ボタンは画面下部にあった方が嬉しいです。

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご意見ありがとうございます。

ヘッダーとフッターに表示されていた要素をまとめ、より執筆に利用できるエリアが広がるように変更いたしました。
https://qiita.com/release-notes/2023-07-11

今後エディタに機能追加をしていきたいと考えているので、見た目をすっきりさせたいという意図もあるのですが

投稿ボタンが上にあるサービスも珍しくない（note、Zennなど）ことと、
実際に執筆エリアが狭いというご意見もいただいていたため実施しました。
（投稿ボタンが下部にあることで、特にiOSで投稿ボタンが画面外に出てしまうバグなどがあったのも開発目線の一因ではあります..）

従来の位置に慣れた方には、不便を感じさせてしまうということは理解しており
様々なご意見を参考に、今後の改善に繋げていきたいと思います🙇

投稿ボタンが目立たなくて見つけづらいのは、ごもっともなので改善します！🙏

**Reply by JunichiIto** [reply]
背景のご説明、どうもありがとうございます。

> 投稿ボタンが上にあるサービスも珍しくない（note、Zennなど）

noteとZennの投稿画面を見てきました。たしかに投稿ボタンは画面の上部にありますね。
ただ、Qiitaと違うのは全体的に画面がシンプルで投稿ボタンの周りに空白が多いため、比較的ボタンの存在が目立ちやすい（良い意味でボタンが浮いている）ように感じました。

<img width="1275" alt="Screenshot 2023-08-03 at 11 42 41" src="https://github.com/increments/qiita-discussions/assets/1148320/c739b670-6c57-4de7-a6d4-87ded204df2f">

Qiitaはボタンの周辺にテキストボックスやアイコンが並んでいて、投稿ボタンが画面の中の"one of them"の存在になっている感があります。
このあたりがnoteやZennとの違いなのかもしれません。

> 投稿ボタンが目立たなくて見つけづらいのは、ごもっともなので改善します！🙏

そうですね、それで多少改善されると良いかなーと思います。

**Reply by tanaka-asami415** [reply]
@JunichiIto 
こちらこそ詳細なご意見をありがとうございます！
今後の改善の参考にさせていただきます🙇

**Reply by tanaka-asami415** [reply]
@JunichiIto 
お待たせいたしました🙇
エディタの公開設定ボタンの色を変更しました！

![image](https://github.com/increments/qiita-discussions/assets/103916185/18155eb2-7cbc-4853-8fee-5eee11358f57)

**Reply by JunichiIto** [reply]
@tanaka-asami415 確認しました。ご対応ありがとうございます！
